meetings-meetups/_posts/2020-03-26-virtual-foss-hours.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: redirect
+redirect: https://fossrit.github.io/
+date-start: "2020-03-26 17:00"
+date-end: "2020-03-26 18:00"
+location: "Online"
+title: "FOSS Hours"
+rrule: "FREQ=WEEKLY;UNTIL=20200423T190000"
+---
+Come meet other students and faculty involved with FOSS efforts at RIT!
+
+https://meet.jit.si/rit-foss-hours (password `foss`)
```

Comment:
Note the URL in the event body text doesn't register as a hyperlink in the Calendar view.

Reply:
Makes sense then. Good to know!